src/validator.directive.js

```diff
@@ -225,6 +228,7 @@
             ctrl.$render();
             if (attrs.messageId) angular.element(document.querySelector('#' + attrs.messageId)).html('');
             else element.next().html('');
+            if ($validationProvider.validCallback) $validationProvider.validCallback(element);
```

Comment:
`validCallback` is used for the input validate as valid's callback. not for reset.